charts/agh2/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.2.1
+version: 2.3.1
```

Comment:
Patch version MUST be reset to 0 when minor version is incremented.

ref: https://semver.org/#spec-item-7